manifests/params.pp

```diff
@@ -157,6 +157,13 @@
   $puppet_group       = 'puppet'
   $puppetdir          = $puppet::params::dir
 
+  # The puppet-agent package, (puppet 4 AIO) doesn't create a puppet group
+  if versioncmp($::puppetversion, '4.0') > 0 {
```

Comment:
@mmoll Then again what's the worst that can happen?  If a non AIO puppet 4 *is* installed and has already created a puppet user/group these superfluous resource declarations won't actually do anything will they?